fuzzers/074-dump_all/jobnodes.tcl

```diff
@@ -13,10 +13,24 @@ set root_fp [open "root_node_${blocknb}.csv" w]
 
 set nodes [get_nodes]
 
-for {set j $start } { $j < $stop } { incr j } {
+create_pblock exclude_roi
```

Comment:
This exclude_roi seems problematic.  Why is it being done?

Reply:
This is a follow up to the usage of exclude ROI in the 005 fuzzer. Tileconn needs to avoid building the tilegrid with the locked tiles as it would conflict with the original tilegrid